packages/build/src/npm-packages/constants.ts

```diff
@@ -1,10 +1,18 @@
 import path from 'path';
 
-export const PLACEHOLDER_VERSION = '0.0.0-dev.0';
 export const PROJECT_ROOT = path.resolve(__dirname, '..', '..', '..', '..');
 export const LERNA_BIN = path.resolve(
   PROJECT_ROOT,
   'node_modules',
   '.bin',
   'lerna'
 );
+
+/** Packages which get bumped only as part of the mongosh release. */
+export const MONGOSH_RELEASE_PACKAGES = ['mongosh', '@mongosh/cli-repl'];
+
+/** Packages which always get ignored when doing a release or bump */
+export const IGNORE_BUMP_PACKAGES = [
```

Comment:
ignoring these since these are private, unless it'd make sense to publish anyways?